lowell-dashboard/app/forms.py

```diff
@@ -74,3 +75,39 @@ class LoginForm(FlaskForm):
 	Create a submit button
 	'''
     submit = SubmitField('Login')
+
+class UpdateAccountForm(FlaskForm):
+    '''
+    Create a string field and call it username
+    Field can't be empty, length of the string must be between 2 and 20 characters
+    '''
+    username = StringField('Username',
+                           validators=[DataRequired(), Length(min=2, max=20)])
+    '''
+    Create a string field and call it email
+    This field will accept email and use the Email validator to make sure it's a email
+    '''
+    email = StringField('Email',
+                        validators=[DataRequired(), Email()])
+    '''
+    Create a file field that will accept a file
+    The validator for this field is a file allower that only accepts files with extensions of jpg and png
+    '''
+    picture = FileField('Update Profile Picture', validators=[FileAllowed(['jpg', 'png'])])
+    '''
+    Create a submit button
+    '''
+    submit = SubmitField('Update')
+
+    def validate_username(self, username):
+        if username.data != current_user.username:
+            user = User.query.filter_by(username=username.data).first()
+            if user:
+                raise ValidationError('That username is taken. Please choose a different one')
+
+    def validate_email(self, email):
```

Comment:
I don't think most websites let you change your email. If we do something like this involving email we should at least do something like confirm email or something.

What will email be used for?

Reply:
Email will be used for logging in. Idk if we want to use that rn, but it's a placeholder. We can change it to username in the future but sometimes I think email is easier to remember. Then again I have multiple email and I forgot which email I used to create the account. Maybe we can use both, username and email for login. Yeah we should ignore the profile images since for some reason it's making new ones every time I update a profile image. But for now it's no problem. I can create a new folder and store the images there and also git ignore it. A lot of website let your change your email it's just that it's hard to since they don't want you making an accidental mistake. We can do it like github and put it at the very bottom and label it "danger zone you're about to make changes to your account." And the account page will be more populated when we add things like classes and other stuff.

Reply:
That works we can change it before release